### PR TITLE
invoice page UI changes and report page icon, UI changes

### PR DESCRIPTION
**Pull request by Ankita012**
## Notion card
https://www.notion.so/6afc9500b2cc42af86be55f37894fb09?v=185bd45b9dc24dfcad9c57c07d32a7c0&p=c87dbabdf1374c5a9c3b7372f3af5a38&pm=s
https://www.notion.so/6afc9500b2cc42af86be55f37894fb09?v=185bd45b9dc24dfcad9c57c07d32a7c0&p=d964a31b05f74812905554d6ef6316b3&pm=s

## Summary

- UI changes in invoice list page
- Fix Icons for Reports

## Preview
Screenshot 2023-01-03 at 7 19 05 PM
Screenshot 2023-01-03 at 7 21 29 PM
Screenshot 2023-01-03 at 7 21 39 PM

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactor (non-breaking and retains same functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

- Check for UI changes in the invoice list page.
- Check for icons and UI on report page

### Checklist:

- [x] I have manually tested all workflows
- [x] I have performed a self-review of my own code
- [x] I have annotated changes in the PR that are relevant to the reviewer
- [ ] I have added automated tests for my code

### CODE REVIEW

**Review by apoorv1316 — CHANGES_REQUESTED** [review]
@Ankita012 Please change the font size to 40 px on the amount. It is 36px currently. Also, there has to be a hover effect on the reports icon. You can see the effect from here https://www.figma.com/proto/4gdPDRPqGqQqPYsO3Qj8CG/Miru-App?page-id=6785%3A69221&node-id=9248%3A102629&viewport=63%2C-7180%2C0.4&scaling=min-zoom
Screenshot 2023-01-04 at 11 51 49 AM

File: app/javascript/images/Revenue.svg [hunk 1]
```diff
@@ -1,8 +1,13 @@
...
-<path fill-rule="evenodd" clip-rule="evenodd" d="M38.9111 20.0153C39.593 20.7946 39.514 21.9792 38.7347 22.6611L28.7347 31.4111C27.9554 32.093 26.7708 32.014 26.0889 31.2347C25.407 30.4554 25.486 29.2708 26.2653 28.5889L36.2653 19.8389C37.0447 19.157 38.2292 19.236 38.9111 20.0153Z" fill="#5B34EA"/>
-<path fill-rule="evenodd" clip-rule="evenodd" d="M13.9111 8.76533C14.593 9.54465 14.514 10.7292 13.7347 11.4111L3.73472 20.1611C2.9554 20.843 1.77085 20.764 1.08894 19.9847C0.407041 19.2054 0.486011 18.0208 1.26533 17.3389L11.2653 8.58894C12.0446 7.90704 13.2292 7.98601 13.9111 8.76533Z" fill="#5B34EA"/>
-<path fill-rule="evenodd" clip-rule="evenodd" d="M1.08894 20.0153C1.77085 19.236 2.9554 19.157 3.73472 19.8389L13.7347 28.5889C14.514 29.2708 14.593 30.4554 13.9111 31.2347C13.2292 32.014 12.0446 32.093 11.2653 31.4111L1.26533 22.6611C0.486011 21.9792 0.407041 20.7946 1.08894 20.0153Z" fill="#5B34EA"/>
+<svg width="77" height="71" viewBox="0 0 77 71" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (prasanthchaduvula) [inline app/javascript/images/Revenue.svg:1]: @Ankita012 Please move these svg icons used in this PR to miruicons

**Review by prasanthchaduvula — CHANGES_REQUESTED** [review]

File: app/javascript/src/components/Reports/reportList/index.tsx [hunk 2]
```diff
@@ -3,21 +3,27 @@ import React from "react";
 
 import ReportCard from "./reportCard";
 
-const calendar = require("../../../../images/Calendar.svg"); // eslint-disable-line
-const hours = require("../../../../images/Hours.svg");
-const overdueOutstandingIcon = require("../../../../images/OverdueOutstanding.svg");
-const revenue = require("../../../../images/Revenue.svg");
+const calendar = require("../../../miruIcons/Calendar.svg");
+const calendarHover = require("../../../miruIcons/CalendarHover.svg");
```
Comment (prasanthchaduvula) [inline app/javascript/src/components/Reports/reportList/index.tsx:7]: @Ankita012 Please change the imports to this below format ```
import {
 CalendarHoverIcon,
  HoursIcon,
  HoursHoverIcon,
  OverdueOutstandingIcon,
  OverdueOutstandingHoverIcon.
  RevenueIcon,
  RevenueHoverIcon

} from "miruIcons";
```

**Review by prasanthchaduvula — CHANGES_REQUESTED** [review]

**Comment by supriya3105** [comment]
@Ankita012 Please update the branch.

File: app/javascript/src/miruIcons/index.ts [hunk 3]
```diff
@@ -38,12 +38,16 @@ import {
   CurrencyCircleDollar,
 } from "phosphor-react";
 
+const revenueHoverIcon = require(".../../../assets/images/RevenueHover.svg");
```
Comment (prasanthchaduvula) [inline app/javascript/src/miruIcons/index.ts:41]: @Ankita012 Create a folder inside miruIcons to store the SVG files and import them inside miruIcons and use them. Delete the SVG files that got added to assets in this PR.

**Review by prasanthchaduvula — CHANGES_REQUESTED** [review]

File: app/javascript/src/miruIcons/index.ts (hunk 3 above)
Comment (Ankita012) [reply app/javascript/src/miruIcons/index.ts]: Done

**Pull request merged**